Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+
+[packages]
+Flask = "*"
+gunicorn = "*"
```

Comment:
Should we add restrict dependency? It will help to avoid issues caused due to breaking changes in dependent packages.

```
[packages]
Flask = "~=1.1.2"
gunicorn = "~=20.0.4"
```

Reply:
makes sense, @beav thoughts?

Reply:
I've seen different projects do it both ways. As long as you remember to update to the latest versions once in awhile (perhaps make a monthly reminder in your calendar), it's OK to pin the versions.

Reply:
> Should we add strict dependency? It will help to avoid issues caused due to breaking changes in dependent packages.
> ```
> [packages]
> Flask = "~=1.1.2"
> gunicorn = "~=20.0.4"
> ```

I think it's good idea to lock these dependencies. As @patilsuraj767 mentioned let's point it to latest.

Reply:
done.